source/css/_global/index.styl

```diff
@@ -102,7 +121,8 @@ table
   th,
   td
     padding: 0.3rem 0.6rem
-    border: 1px solid $light-grey
+    border: 1px solid #dfe2e5
```

Comment:
The color here, you'd better use the color in the `var.styl`.

Reply:
If you think $light-grey is so light, then you can use the stylus built-in function `darken()`, such as `darken($light-grey, 10%)`